app/routines/update_user_salesforce_info.rb

```diff
@@ -113,6 +113,8 @@ def call
               :pending_faculty
             end
 
+          user.using_openstax = leads.map[&:adoption_status].uniq
```

Comment:
`leads.map[&:adoption_status].uniq` returns an array, but ` user.using_openstax` needs a string.

Reply:
I really don't know which lead you should store in/as `using_openstax`. I _think_ it would something similar as we're doing with `user.faculty_status` right above.